E2E-Demos/Question-Answering/02.serve-vectorstore.ipynb

```diff
@@ -137,8 +137,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "predictor_image = (input(\"Enter the name of the predictor image (default: gcr.io/mapr-252711/ezua-demos/vectorstore:v0.1.0): \")\n",
-    "                   or \"gcr.io/mapr-252711/ezua-demos/vectorstore:v0.1.0\")"
+    "predictor_image = (input(\"Enter the name of the predictor image (default: dpoulopoulos/vectorstore:b98a87f): \")\n",
```

Comment:
why are we using personal repo?

Reply:
ok, so public. Then can we retag it to `v0.1.1`?

Reply:
Maybe set DEFAULT_PREDICTOR_IMAGE='dpoulopoulos/vectorstore:v0.1.1' in the beginning of the file?